FrameworkSystem/private/logging/Logger.py

```diff
@@ -147,7 +147,7 @@ def getAllPossibleLevels( self ):
   def shown( self, levelName ):
```

Comment:
Yes, I use Logger.py in my tests, and I use _shown_ but I notice a problem with the method, so I corrected it.